CMakeLists.txt

```diff
@@ -53,15 +53,29 @@ if(OT_PACKAGE_VERSION STREQUAL "")
 endif()
 message(STATUS "Package Version: ${OT_PACKAGE_VERSION}")
 
-set(OT_PLATFORM_LIB "openthread-efr32-soc")
-set(OT_PLATFORM_LIB_RCP openthread-efr32-rcp)
+# ==============================================================================
+# Set platform libs and allow overriding
+# ==============================================================================
+if(NOT OT_PLATFORM_LIB)
+    set(OT_PLATFORM_LIB "openthread-efr32-soc" CACHE STRING "default efr32 SoC platform library" FORCE)
+endif()
+
+if(NOT OT_PLATFORM_LIB_RCP)
+    set(OT_PLATFORM_LIB_RCP "openthread-efr32-rcp" CACHE STRING "default efr32 RCP platform library" FORCE)
+endif()
+
+if(NOT OT_MBEDTLS_RCP)
+    set(OT_MBEDTLS_RCP "openthread-efr32-rcp-mbedtls" CACHE STRING "default efr32 RCP mbedtls library" FORCE)
+endif()
 
 set(OT_PLATFORM "external" CACHE STRING "disable example platforms" FORCE)
 
 # ==============================================================================
 # mbedtls
 # ==============================================================================
-set(OT_EXTERNAL_MBEDTLS "openthread-efr32-soc-mbedtls" CACHE STRING "use silabs mbedtls" FORCE)
+if(NOT OT_EXTERNAL_MBEDTLS)
```

Comment:
I know there are commits where you say you are combining the mbedtls with the platform lib builds.  Is this why there's no SoC library for mbedtls like there is for RCP?  Why do we treat RCP differently than SoC when it comes to how we build mbedtls?

Reply:
We don't combine `mbedtls with the platform lib builds`. We're combining the slc-generation of the CMake files for these targets. They are still build as two separate libraries. 

There's no difference in behavior between RCP and SoC. In both cases, we are generating 2 CMake files,
- `build/brdXXXXy/slc/(rcp|soc)/CMakeLists.txt` - The platform library target
- `build/brdXXXXy/slc/(rcp|soc)/openthread-efr32-(rcp|soc)-mbedtls.cmake` - The mbedtls library target